src/parsers/RealParser.cpp

```diff
@@ -277,6 +277,9 @@ void qc::QuantumComputation::readRealGateDescriptions(std::istream& is, int line
                 case Teleportation:
                     std::cerr << "Operation with invalid type " << gate << " read from real file. Proceed with caution!" << std::endl;
                     break;
+                default:
+                    std::cerr << "Unknown operation:  " << gate << "!" << std::endl;
+                    break;
```

Comment:
Remove the `case Compound:` ... `case Teleportation:` cases and simply replace them with `default:`.
Maybe this even fixes the Code Coverage 🤷🏼‍♂️

Reply:
Removed the lines as suggested.